ci-operator/step-registry/upi/conf/vsphere/platform-external/upi-conf-vsphere-platform-external-commands.sh

```diff
@@ -94,6 +94,13 @@ export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${RELEASE_IMAGE_LATEST}
 # Ensure ignition assets are configured with the correct invoker to track CI jobs.
 export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}
 
+echo "$(date -u --rfc-3339=seconds) - Discovering controller image 'vsphere-cloud-controller-manager' from release [${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE-}]"
+
+PULL_SECRET="/var/run/secrets/ci.openshift.io/cluster-profile/pull-secret"
```

Comment:
nit: alternatively you can use the global env var: `"${CLUSTER_PROFILE_DIR}"/pull-secret`

Reply:
oh, let me try that.  thanks!